pilot/pkg/bootstrap/server.go

```diff
@@ -211,6 +213,13 @@ func NewServer(args *PilotArgs, stopCh <-chan struct{}) (*Server, error) {
 		return nil, fmt.Errorf("sidecar injector: %v", err)
 	}
 
+	// Run the SDS signing server.
+	// RunCA() must be called after createCA() and initDNSListener()
+	// because it depends on the following conditions:
+	// 1) CA certificate has been created.
+	// 2) grpc server has been generated.
+	s.RunCA(s.secureGRPCServerDNS, s.ca, caOpts, stopCh)
```

Comment:
nit: now that we have it structured this way, can we can use addStartFunc and not have to pass the stopCh?

I am not sure if the `addStartFunc` pattern is a good idea, but if others are using it we probably should too for consistency?

Reply:
Done.